React/app/containers/CharityDetailPage/index.js

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Maybe try to make this a class? It would be beneficial for when we need to make axios calls to pull data from db